SpatialGDK/Documentation/content/getting-started.md

```diff
@@ -23,3 +20,5 @@ Visit the **feedback** section in our [forums](https://forums.improbable.io/) an
 
 **GitHub issues**</br>
 Create an issue in [this repository](https://github.com/spatialos/UnrealGDK/issues).
+
+#### Next: [Set Up Prerequisites]({{urlRoot}}/content/get-started/prerequisites.md) 
```

Comment:
Set up prerequisites 

(note case)

BUT I am not sure that's the next title? It needs to be consistent at the bottom of the page with the title at the top.

Reply:
yea sorry that is on the dupe version of the getting started page which i now deleted